streaming/stream_session.cc

```diff
@@ -111,24 +112,30 @@ void stream_manager::init_messaging_service_handler(abort_source& as) {
             return make_ready_future<>();
         });
     });
-    ms.register_stream_mutation_fragments([this, &as] (const rpc::client_info& cinfo, streaming::plan_id plan_id, table_schema_version schema_id, table_id cf_id, uint64_t estimated_partitions, rpc::optional<stream_reason> reason_opt, rpc::source<frozen_mutation_fragment, rpc::optional<stream_mutation_fragments_cmd>> source) {
+    ms.register_stream_mutation_fragments([this, &as] (const rpc::client_info& cinfo, streaming::plan_id plan_id, table_schema_version schema_id, table_id cf_id, uint64_t estimated_partitions,
+            rpc::optional<stream_reason> reason_opt,
+            rpc::optional<service::session_id> session,
```

Comment:
So I wonder how is this different from the `plan_id`? As far as I know it is also a per "streaming session" uuid. Aren't we trying to invent here something that already exist? @asias

Reply:
The plan_id is per stream_plan. We can use multiple stream_plans to stream data under a single "streaming session" to move data.